libs/template/themes/icons/mappers.ts

```diff
@@ -0,0 +1,26 @@
+import { IconMapper } from '@spryker-oryx/experience';
+import { IconTypes } from './icon.model';
+
+export const materialMapper: IconMapper = {
+  id: 'material-icons',
+  styles: `
+    font-family: 'Material Icons';
+    font-weight: normal;
+    font-style: normal;
+    font-size: 24px;
+  `,
+  mapping: {
+    [IconTypes.Cart]: 'shopping_cart_checkout',
+  },
+};
+
+export const fontAwesomeMapper: IconMapper = {
+  id: 'fa',
+  styles: `
+    font-family: 'Font Awesome 6 Free';
+    font-weight: 900;
```

Comment:
```suggestion
    font-family: 'Font Awesome 6 Free';
    font-weight: 900;
    font-size: calc(var(--oryx-icon-size, 24px) * 0.7);
    align-self: center;
```

Let's calc the font-size for now, more investigation is needed to see how FA, MD and Spryker icon sizes align.